Yoast/YoastSEO.js
# Add singleH1Assessment, remove urlLength and urlStopwords in Recalibration

## Summary

This PR can be summarized in the following changelog entry:

* Deprecates two SEO assessments that check the length of the URL and whether stop words are used in the URL
* Adds an assessment that checks if the article has spurious uses of `<h1>`.

## Relevant technical choices:

* The SingleH1Assessment was transferred from the PR where it was introduced #1406. It makes use of the current HTML parser, making it run another time to pick up `<h1>` tags. @atimmer do you find it acceptable to run HTML parser in this way? Or would you suggest another solution?

## Test instructions

This PR can be tested by following these steps:

* Use the new example to compare regular and recalibration analysis.
* In the regular analysis, check if everything works as before:
  * Add a text with two H1s, no additional assessments should appear
  * Add a slug with a stop word, a urlStopWords assessment should trigger
  * Add a long slug, a urlLength assessment should trigger
* In the recalibration analysis:
  * Add a text with two H1s, a `Single title` assessment should trigger.
  * Add a text with a H1 at the beginning of the text. The `Single title` assessment shouldn't trigger.
  * Add a text with an H1 anywhere but the beginning of the title. The `Single title` should trigger.
  * Add a slug with a stop word, a urlStopWords assessment should **not** trigger
  * Add a long slug, a urlLength assessment should **not** trigger

Fixes #1926 Fixes #1927 Fixes #1928
